src/tests/cmocka/test_ipa_subdomains_server.c

```diff
@@ -465,19 +465,19 @@ static void test_ipa_server_create_trusts_twoway(struct tevent_req *req)
 
     /* Two-way trusts should use the system realm */
     assert_trust_object(test_ctx->ipa_ctx->server_mode->trusts,
```

Comment:
What does this hunk have to do with the rest of the patch? (Also, this test is failing for me after applying the patches)

Reply:
I guess we use different libldb versions. I modified the test so that the results can come in any order.